docs/contents/components/data-display/table/accessibility.mdx

```diff
@@ -0,0 +1,43 @@
+package_name: "@yamada-ui/table"
+order: 3
+with_description: true
+tab: Accessibility
+---
+
+The `Table` follows the [WAI-ARIA - Table Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/table/) for accessibility.
+
+Setting an `aria-label` on the `Table` allows it to be read by screen readers.
+
+```tsx live=false
+<Table columns={columns} data={data} aria-label="DRAGON BALL characters" />
+```
+
+## Keyboard Navigation
+
+| Key             | Description                                                                                              | State |
+| --------------- | -------------------------------------------------------------------------------------------------------- | ----- |
+| `Tab`           | Focuses the first item when focus moves to the table. Focuses the next item if already within the table. | -     |
+| `Shift` + `Tab` | Focuses the previous item that is not disabled.                                                          | -     |
+| `ArrowLeft`     | Focuses the previous item that is not disabled.                                                          | -     |
+| `ArrowRight`    | Focuses the next item that is not disabled.                                                              | -     |
+| `ArrowUp`       | Focuses the previous item that is not disabled.                                                          | -     |
+| `ArrowDown`     | Focuses the next item that is not disabled.                                                              | -     |
+| `Home`          | Focuses the first item that is not disabled.                                                             | -     |
+| `End`           | Focuses the last item that is not disabled.                                                              | -     |
+
+## ARIA Roles and Attributes
+
+| Component                   | Role and Attributes | Usage                                                                                                   |
+| --------------------------- | ------------------- | ------------------------------------------------------------------------------------------------------- |
+| `Table`                     | `role="table"`      | Indicates that the element is a table.                                                                  |
+|                             | `aria-disabled`     | Set to `true` if `disabledRowIds` is specified.                                                         |
+|                             | `aria-rowcount`     | Indicates the total number of rows in the table.                                                        |
+|                             | `aria-colcount`     | Indicates the total number of columns in the table.                                                     |
+| `Thead` <Tag>Internal</Tag> | `aria-sort`         | Set to `ascending` for ascending order, `descending` for descending order, and `none` if not specified. |
```

Comment:
Change `Thead` to `Th`.

Reply:
Fixed.
https://github.com/yamada-ui/yamada-ui/pull/3038/files#diff-7ad307d16404c4e1c0bfc2852ffc14f4d55552661d2b9e66b357cee2c6012757R38-R40